nf_core/__main__.py

```diff
@@ -31,11 +32,26 @@
     "nf-core": [
         {
             "name": "Commands for users",
-            "commands": ["list", "launch", "create-params-file", "download", "licences", "tui"],
+            "commands": [
+                "list",
+                "launch",
+                "create-params-file",
+                "download",
+                "licences",
+                "tui",
+            ],
         },
         {
             "name": "Commands for developers",
-            "commands": ["create", "lint", "modules", "subworkflows", "schema", "bump-version", "sync"],
+            "commands": [
+                "create",
+                "lint",
+                "modules",
+                "subworkflows",
+                "schema",
+                "bump-version",
```

Comment:
Need to add the new command here so that it's grouped properly in the help text.

```suggestion
                "create-logo",
                "bump-version",
```